tests/pgautofailover_utils.py

```diff
@@ -284,6 +304,10 @@ def create(self):
                              name="keeper init",
                              timeout=COMMAND_TIMEOUT)
 
+        if (self.get_state() == 'primary'):
```

Comment:
Now that is strange. Certainly we want to be able to authenticate the `pgautofailover_monitor` user before we get to be primary, like in WAIT_PRIMARY stage already? Also, what about monitoring the secondary? Feels strange.

Reply:
removed

Reply:
because it was not needed anymore after moving password setting responsibility to test